### PR TITLE
Deprecated the croud consumers commands

### PR DESCRIPTION
**Pull request by Taliik**
## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -5,6 +5,9 @@ Changes for croud
 Unreleased
 ==========
 
+- Deprecated the ``croud consumers`` commands ``deploy``, ``list``,
+  ``edit`` and ``delete``, as they will be removed in the future.
```
Comment (mechanomi) [inline CHANGES.rst:9]:
```suggestion
  ``edit``, and ``delete``, as they will be removed in the future.
```

Comment (mechanomi) [reply CHANGES.rst:9]: might also make sense to add notices to the user-facing docs too?

Comment (MarkusH) [reply CHANGES.rst:9]: Maybe even remove them already?

File: docs/commands/consumers.rst [hunk 2]
```diff
@@ -2,6 +2,11 @@
 ``consumers``
 =============
 
+.. warning::
+
+    The consumers commands are deprecated and will be removed in the future.
```
Comment (mechanomi) [inline docs/commands/consumers.rst:7]:
```suggestion
    The ``consumers`` commands are deprecated and will be removed in the future.
```

File: docs/commands/consumers.rst [hunk 3]
```diff
@@ -14,6 +19,10 @@ The ``consumers`` command lets you manage consumer products.
 ``consumers deploy``
 ====================
 
+.. warning::
+
+    The consumers deploy command is deprecated.
```
Comment (mechanomi) [inline docs/commands/consumers.rst:24]:
```suggestion
    The ``consumers deploy`` command is deprecated.
```

File: docs/commands/consumers.rst [hunk 4]
```diff
@@ -49,6 +58,10 @@ Example
 ``consumers list``
 ==================
 
+.. warning::
+
+    The consumers list command is deprecated.
```
Comment (mechanomi) [inline docs/commands/consumers.rst:63]:
```suggestion
    This command is deprecated and will be removed in the future.

```

File: docs/commands/consumers.rst [hunk 5]
```diff
@@ -71,6 +84,10 @@ Example
 ``consumers edit``
 ==================
 
+.. warning::
+
+    The consumers edit command is deprecated.
```
Comment (mechanomi) [inline docs/commands/consumers.rst:89]:
```suggestion
    This command is deprecated and will be removed in the future.
```

File: docs/commands/consumers.rst [hunk 6]
```diff
@@ -95,6 +112,10 @@ Example
 ``consumers delete``
 ====================
 
+.. warning::
+
+    The consumers delete command is deprecated.
```
Comment (mechanomi) [inline docs/commands/consumers.rst:117]:
```suggestion
    This command is deprecated and will be removed in the future.
```

**Pull request merged**